product_docs/docs/pgd/4/deployments/tpaexec/using_tpaexec.mdx

```diff
@@ -0,0 +1,143 @@
+
+### Common configuration options
+
+Other configuration options include:
+
+#### Owner
+Every cluster must be directly traceable to a person responsible for the provisioned resources.
+
+By default, a cluster is tagged as being owned by the login name of the user running `tpaexec provision`. If this name does not identify a person (for example, `postgres`, `ec2-user`), you must specify `--owner SomeId` to set an identifiable owner.
+
+You may use your initials, or "Firstname Lastname", or anything else that identifies you uniquely.
+
+#### Platform options
+The default value for `--platform` is `aws`. It is the platform supported by the BDR-Always-ON architecture. 
+
+Specify `--region` to specify any existing AWS region that you have access to (and that permits the required number of instances to be created). The default region is eu-west-1.
+
+Specify `--instance-type` with any valid instance type for AWS. The default is t3.micro. 
+
+### Subnet selection
+By default, each cluster is assigned a random /28 subnet under 10.33/16, but depending on the architecture, there may be one or more subnets, and each subnet may be anywhere between a /24 and a /29.
+
+Specify `--subnet` to use a particular subnet. For example, `--subnet 192.0.2.128/27`.
+
+Alternatively, specify `--subnet-pattern` to generate random subnets (as many as required by the architecture) matching the given pattern. For example, `--subnet-pattern 192.0.x.x`.
+
+### Disk space
+Specify `--root-volume-size` to set the size of the root volume in GB. For example, `--root-volume-size 64`. The default is 16GB. (Depending on the image used to create instances, there may be a minimum size for the root volume.)
+
+For architectures that support separate postgres and barman volumes:
+
+Specify `--postgres-volume-size` to set the size of the Postgres volume in GB. The default is 16GB.
+
+Specify `--barman-volume-size` to set the size of the Barman volume in GB. The default is 32GB.
+
+### Distribution
+Specify `--os` or `--distribution` to specify the OS to be used on the cluster's instances. The value is case-sensitive.
+
+The selected platform determines which distributions are available and which one is used by default. For more details, see `tpaexec info platforms/<platformname>`.
+
+In general, you can use "Debian", "RedHat", and "Ubuntu" to select TPA images that have Postgres and other software preinstalled (to reduce deployment times). To use stock distribution images instead, append "-minimal" to the value, for example, `--distribution Debian-minimal`.
+
+### 2ndQuadrant repositories
+By default, TPAexec installs the 2ndQuadrant public repository and adds on any product repositories that the architecture requires.
+
+Specify `--2Q-repositories source/name/maturity … ` to specify the complete list of 2ndQuadrant repositories to install on each instance in addition to the 2ndQuadrant public repository.
```

Comment:
I think for consumption by customers maturity should be always `release` so I'd just put the `release` here rather than maturity